docs/source/asr/speaker_diarization/intro.rst

```diff
@@ -0,0 +1,70 @@
+Speaker Diarization
+==================================
+
+Speaker Diarization (SD) is the task of segmenting audio recordings by speaker labels, that is Who Speaks When?
+
+.. image:: images/sd_pipeline.png
+        :align: center
+        :alt: Speaker Diarization pipeline [todo]
+        :scale: 50%
+
+
+A diarization system consists of **Voice Activity Detection (VAD)** model to get the time stamps of audio where speech is being spoken ignoring the background and 
+**Speaker Embeddings** model to get speaker embeddings on segments that were previously time stamped by VAD. 
+These speaker embeddings would then be clustered into clusters based on number of speakers present in the audio recording.
+
+In NeMo we support both **oracle VAD** and **non-oracle VAD** diarization. 
+
+``Quick Start``
+
+.. note::
+  For model details and deep understanding, please refer to ``<NeMo_git_root>/tutorials/speaker_recognition/Speaker_Diarization_Inference.ipynb``.
+
+        
+Run and evaluate speaker diarizer with below command:
+
+.. code-block:: bash
+
+  python examples/speaker_recognition/speaker_diarize.py \
+  diarizer.paths2audio_files=<either list of audio file paths or file containing paths to audio files> \
+  diarizer.path2groundtruth_rttm_files=<(Optional) either list of rttm file paths or file containing paths to rttm files> \
+  diarizer.vad.model_path="vad_telephony_marblenet" \
+  diarizer.vad.threshold=0.7 \
```

Comment:
This is a duplicate of your speaker_diarizer.py docstring no? Cant we just refer to it and avoid this. Future updates will break this codeblock - so its double work to maintain it.

Reply:
ouch, we don't have good instruction in speaker_diarize.py 
@nithinraok  shall we update it and move it?

Reply:
Move it, we shall update the function call and comeback to this again after thats done